proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -463,11 +507,17 @@ def build(self):
             self.deltaP = Reference(self.feed_side.deltaP)
 
         # mass transfer
+        mass_transfer_phase_comp_initialize = {}
+        for t in self.flowsheet().config.time:
+            for j in self.config.property_package.component_list:
+                    mass_transfer_phase_comp_initialize[(t, 'Liq', j)] = \
+                        value(self.feed_side.properties_in[t].get_material_flow_terms('Liq', j)
+                              * self.recovery_mass_phase_comp[t, 'Liq', j])
```

Comment:
This one could be functionalized as well, though it's obviously not as clean.

Reply:
Oops good catch, that variable was in another section.